Sources/MongoSwift/ClientSession.swift

```diff
@@ -124,11 +96,11 @@ public final class SyncClientSession {
     /// The options used to start this session.
     public let options: ClientSessionOptions?
 
-    /// Initializes a new client session.
-    internal init(client: SyncMongoClient, options: ClientSessionOptions? = nil) throws {
+    /// This type is not meant to be instantiated directly. Should only be instantiated via subclasses.
+    // swiftformat:disable:next redundantFileprivate
```

Comment:
this seems to be a swiftformat bug, opened https://github.com/nicklockwood/SwiftFormat/issues/493